### ISSUE TITLE
Populate secret and/or config stores from environment variables

### ISSUE THREAD

**Issue by joeshaw** [fastly/Viceroy#359]
You can populate config or secret stores either inline in the fastly.toml file or by reading a file from disk.  It would be nice if you could also populate them with environment variables.

My current use case for this is passing in an API credential when running Viceroy in a test runner mode.  I don't want to check this credential into git, and I would rather not have to bother with an external, local-only file.

**Comment by cee-dub** [comment]
This is sort of the opposite direction of what I'd like to do, which is to populate environment variables within the process space from (probably) a config store.

**Comment by joeshaw** [comment]
Yes, I don't view the two as mutually exclusive.  And depending on how we namespace things, it's maybe possible that an environment variable `FOO_BAR=baz` in the host environment is _also_ `FOO_BAR=baz` in the guest environment.

**Comment by cee-dub** [comment]
Unintentional if it sounded like I was saying these were in conflict or this wouldn't also be very useful. It's a good fit for a tool like Viceroy.
